File: app/src/main/java/com/mya/games/colipop/ResourceUtils.java
package com.mya.games.colipop;

import android.graphics.Bitmap;

public class ResourceUtils {

    public static void recicleBitmaps(Bitmap[] bitmaps) {
        if (bitmaps == null) {
            return;
        }
        for (Bitmap bitmap : bitmaps) {
            if (bitmap != null) {
                bitmap.recycle();
            }
        }
    }

}

File: app/src/main/java/com/mya/games/colipop/ResourceUtils.kt
package com.mya.games.colipop

import android.graphics.Bitmap

object ResourceUtils {

    fun recycleBitmaps(bitmaps: Array<Bitmap>?) {
        if (bitmaps == null) {
            return
        }
        for (bitmap in bitmaps) {
            bitmap.recycle()
        }
    }

}
